src/app/domain/accounts/models.py

```diff
@@ -26,7 +26,8 @@ class User(orm.DatabaseModel, orm.AuditColumns):
     is_superuser: Mapped[bool] = mapped_column(default=False)
     is_verified: Mapped[bool] = mapped_column(default=False)
     verified_at: Mapped[datetime | None]
-    joined_at: Mapped[datetime] = mapped_column(server_default=func.now())
+    created: Mapped[datetime] = mapped_column(server_default=func.now())
+    updated: Mapped[datetime] = mapped_column(server_default=func.now(), server_onupdate=func.now())
```

Comment:
Instead of adding these here, can you add the the `app.lib.db.orm.AuditColumns` mixin instead?  This is a re-export of the 2.0 functionality.  We can open a PR to Starlite if you'd like to see the `server_onupdate` portion added.  We don't currently have that because a `server_onupdate` fires every time an update statement executes where as the implementation in starlite only fires the model is "dirty", but I'm open to changing that.